os/sys/process.c

```diff
@@ -301,7 +288,7 @@ do_event(void)
   }
 }
 /*---------------------------------------------------------------------------*/
-int
+process_num_events_t
```

Comment:
As i understand C - operator + returns int value.
You down-converts it into u8 - that takes more code and cpu cycles

Reply:
poll_requeted should be same type as nevents - to avoid this penalty. but you use bool type - that is int sized